services/terraformer/server/server.go

```diff
@@ -151,7 +172,23 @@ func (*server) DestroyInfrastructure(ctx context.Context, req *pb.DestroyInfrast
 				// Key under which the state file is stored in minIO.
 				key := fmt.Sprintf("%s/%s", projectName, c.Id())
 
-				return mc.RemoveObject(ctx, "claudie-tf-state-files", key, minio.RemoveObjectOptions{
+				// Key under which the lockfile id is stored in dynamodb
+				dynamoLockId, err := attributevalue.Marshal(
+					fmt.Sprintf("%s/%s/%s-md5", minioBucket, projectName, c.Id()),
+				)
+				if err != nil {
+					log.Warn().Msgf("Error composing state lockfile id for cluster %s : %s", c.Id(), config.Name)
+				}
+				fmt.Println(dynamoLockId)
```

Comment:
I just noticed this print. Do we need it? If so, can we make it `log.Debug().Msgf("some message")`?

Reply:
This signals when the `Sprintf` output could not be marshalled to DynamoDB request. I think we can make it `debug`